src/Microsoft.Health.Fhir.Core/Data/OperationDefinition/includes.json

```
@@ -25,7 +25,7 @@
             "use": "in",
             "min": 0,
             "max": "1",
-            "documentation": "The number of 'include' resources to retrieve. Defaults to 500.",
+            "documentation": "The number of 'include' resources to retrieve. Defaults to 1000.",
```

Since this is configurable, thinking we remove the default text here.

```suggestion
            "documentation": "The number of 'include' resources to retrieve.",
```

---

Fixed, as suggested.